portal/src/main/webapp/WEB-INF/jsp/global/global_variables.jsp

```diff
@@ -170,17 +169,13 @@
 <script>
 
     $(document).ready(function() {
-        window.sessionServiceUrl = '<%=sessionServiceUrl%>';
-        if(window.sessionServiceUrl) {
-            window.sessionServiceUrl = window.sessionServiceUrl.replace(/http(s)*:\/+/, '');
-        }
         var getCohortName = function(){
             var def = new $.Deferred();
             if (window.isVirtualStudy) {
                 if (window.cohortIdsList.length === 1) {
                     $.ajax({
                         method: 'GET',
-                        url: 'proxy/' + window.sessionServiceUrl + 'virtual_study/' + window.cohortIdsList[0]
+                        url: 'api-legacy/proxy/session-service/virtual_study/' + window.cohortIdsList[0]
```

Comment:
@kalletlak why do we have to rely on legacy proxy?

Reply:
all the session service related endpoints are under `api-legacy/`

Reply:
@kalletlak I see, we shouldn't rely on `api-legacy` for anything new

Reply:
@kalletlak Unless there's actually a reason to use `api-legacy/` over `api/` because there is some difference in functionality. Then we should fix `api/` to handle that

Reply:
Sure! we could do it in another PR.

Reply:
@kalletlak sounds good, thanks!